meetings/2021-11-22.md

```diff
@@ -0,0 +1,93 @@
+* Call: https://meet.jit.si/solid-authentication
+* Chat: https://gitter.im/solid/authentication-panel
+* Repository: https://github.com/solid/authentication-panel
+
+## Present
+* Aaron Coburn
+* Nicolas AS
+* e Pavlik
+
+## Regrets
+* Pete Edwards (unless there is something specific to testing on the agenda)
+
+---
+
+## Announcements
+
+### Meeting Recordings and Transcripts
+* No audio or video recording, or automated transcripts without consent. Meetings are transcribed and made public. If consent is withheld by anyone, recording/retention must not occur.
+* Use panel chat and repository. Queue in call to talk.
+
+
+### Participation and Code of Conduct
+* [Join the W3C Solid Community Group](https://www.w3.org/community/solid/join), [W3C Account Request](http://www.w3.org/accounts/request), [W3C Community Contributor License Agreement](https://www.w3.org/community/about/agreements/cla/)
+* [Solid Code of Conduct](https://github.com/solid/process/blob/master/code-of-conduct.md), [Positive Work Environment at W3C: Code of Ethics and Professional Conduct](https://github.com/solid/process/blob/master/code-of-conduct.md)
+* If this is your first time, welcome! please introduce yourself.
+
+
+### Scribes
+* Nicolas AS
+* e Pavlik
+
+---
+
+## Agenda
+
+* [Open Pull Requests](https://github.com/solid/solid-oidc/pulls)
+  * [Add RS associated AS issuing Access Token PR#18](https://github.com/solid/solid-oidc/pull/18)
+* [Specify the client authentication method when using a Client Identifier](https://github.com/solid/authentication-panel/issues/201)
+
+---
+
+## Topics
+
+### Open Pull Requests
+
+https://github.com/solid/solid-oidc/pulls
+
+#### [Add RS associated AS issuing Access Token PR#18](https://github.com/solid/solid-oidc/pull/18)
+
+* eP: /Shares latest sequence diagram/. The changes include adding emojis to identify actors. The moment the client is authenticated is made explicit in the diagram too. The change also include removing the textual description of the auth code flow to point to the primer. Access token has bee replaced with ID token. Accordingly, the audience has been moved from solid (for Access token) to client ID (for ID token). The claim in the ID token associated to the client ID is proposed to be `azp`. The binding of the ID token to a DPoP key is also mentioned.
+* AC: I've look into the `azp` claim in the past. It is a good thing to have. I do not recall why we decided to use it. It seems like a good fit, so we should look more into it. It looks like it would desambiguate the claims we use.
+* eP: If we want to issue ID tokens scoped to a specific AuthZ server, then this information would be in the `aud` claim, and it would be good not to rely on this claim to discover the client ID.
+* eP: The PR does not include the discovery of the AuthZ server. Each actor presened in the diagram should be a conformance class so that they are individually described by the spec, in a slightly different structure. 
```

Comment:
```suggestion
* eP: The PR does not include the discovery of the AuthZ server. Each actor presented in the diagram should be a conformance class so that they are individually described by the spec, in a slightly different structure. 
```